packages/dot-merlin-reader/dot-merlin-reader.4.0/opam

```diff
@@ -10,7 +10,7 @@ build: [
   ["dune" "build" "-p" name "-j" jobs]
 ]
 depends: [
-  "ocaml" {> "4.11.1" }
+  "ocaml" {>= "4.11.0"}
```

Comment:
Good catch thanks.

Actually, this could be relaxed even further, down to 4.05.0 …  but not sure there's a point.